src/protos/io/defang/v1/fabric.proto

```diff
@@ -333,6 +337,7 @@ message Service {
   string static_files = 15; // x-defang-static-files: folder with static files
                             // to serve; TODO: not part of spec
   Network networks = 16; // currently only 1 network is supported
+  map<string, StringValue> environment_vars = 17; // 
```

Comment:
Defining a map strings to a StringValue so that we can enable a mapping of keys to nulls.